entrypoint.sh

```diff
@@ -39,6 +39,16 @@ if [[ $LOG_LEVEL ]]; then
     sed -i -e"s/^.*log_level:.*$/log_level: ${LOG_LEVEL}/" /etc/dd-agent/datadog.conf
 fi
 
+if [[ $DD_LOGS_STDOUT ]]; then
+  export LOGS_STDOUT=$DD_LOGS_STDOUT
+fi
+
+if [[ $LOGS_STDOUT == "yes" ]]; then
+  sed -i -e "/^.*_logfile.*$/d" /etc/dd-agent/supervisor.conf
+  sed -i -e "/^.*\[program:.*\].*$/a stdout_logfile=\/dev\/stdout\nstdout_logfile_maxsize=0\nstderr_logfile=\/dev\/stderr\nstderr_logfile_maxsize=0" /etc/dd-agent/supervisor.conf
+  sed -i -e "$ a disable_file_logging: yes" /etc/dd-agent/datadog.conf
```

Comment:
Yes you make a good point. What would be the best way to do this then ? Should I add a second logger in the collector, forwarder, ... that would log to stdout, or should I write a log forwarder that would follow the log files and forward the entries to stdout, or is there another way ?